core/src/test/java/hudson/util/io/TarArchiverTest.java

```diff
@@ -171,6 +146,30 @@ public void emptyDirectory() throws Exception {
         t1.join();
     }
 
+    @Issue("JENKINS-73837")
+    @Test
+    public void emptyDirectory() throws Exception {
```

Comment:
can you reduce the diff my moving this test to the same location as before please

Reply:
I have moved it as seen [here](https://github.com/jenkinsci/jenkins/blob/5340b071277e1a81a77803454440fcf1d5000f72/core/src/test/java/hudson/util/io/TarArchiverTest.java#L130) however it didn't update in the pull request properly.